src/routing/Page.tsx

```diff
@@ -19,18 +19,34 @@ export default function Page(props: PageProps): JSX.Element {
       <h3>{props.title}</h3>
       <Form>
         {props.children}
-        <button
-          className="btn"
-          onClick={(event) => {
-            event.preventDefault();
-            void submitForm();
-            // then submit form has succeeded - add better promise validation here
-            navigate(`${props.nextPage}`);
-          }}
-        >
-          {' '}
-          Next
-        </button>
+
+        {props.prevPage && (
+          <button
+            className="btn prev"
+            onClick={(event) => {
+              event.preventDefault();
+              void submitForm();
+              navigate(props.prevPage as To);
```

Comment:
I don't quite understand the use of `To` here, could you clarify why you used that compared to just `navigate(props.prevPage)`?